README.md

```diff
@@ -94,6 +94,8 @@ You also have to provide the following environment variables from an account tha
 - `RESINTEST_REGISTER_PASSWORD`: The password of the account to register.
 - `RESINTEST_REGISTER_USERNAME`: The username of the account to register.
```

Comment:
This needs brief instructions about how to add billing to your test account, and what that means in practice (the paid tests will be skipped without it).

Ohh, interesting idea that fits with the 'how do we run free tests everywhere' point: should we actually just have devs add an optional `RESINTEST_RECURLY_TOKEN_ID` here? Could we use that to set up and easily turn off/on the billing info in the tests?